airbyte-integrations/connectors/source-google-search-console/source_google_search_console/streams.py

```diff
@@ -123,7 +124,31 @@ def stream_slices(
 
         for site_url in self._site_urls:
             for search_type in self.search_types:
-                yield {"site_url": site_url, "search_type": search_type}
+                start_date = self._get_start_date(stream_state, site_url, search_type)
+                end_date = self._get_end_date()
+
+                if start_date > end_date:
+                    yield from [
```

Comment:
You can use yield instead of yield from list

Reply:
I removed this yield from.